src/index.js

```diff
@@ -1 +1,45 @@
+import Leaderboard from './js/leaderboard.js';
 import './style.css';
+
+const addForm = document.querySelector('form');
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#score');
+const refresh = document.querySelector('button');
+const list = document.querySelector('.score-list');
+
+const leaderboard = new Leaderboard();
+
+let gameId;
+function startGame() {
+  leaderboard
+    .startGame('Term Project')
+    .then((response) => response.result.split(' '))
+    .then((res) => {
+      [gameId] = [res[3]];
+    });
+}
+
+function addToUI(arr) {
+  list.innerHTML = '';
+  arr.forEach((el) => {
+    list.innerHTML += `
+    <li class="item">${el.user} : ${el.score}</li>
+    `;
+  });
+}
+
```

Comment:
- [ ] Please use the function to arrow function. 👉 see here [requirement arrow function](https://user-images.githubusercontent.com/61562626/127013730-08f3b34e-e360-4b80-869b-7e83c7c09340.jpeg)

Reply:
I changed my functions